inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/extract_image_patches.cpp

```diff
@@ -21,17 +21,10 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
     InferenceEngine::Precision::U8,
     InferenceEngine::Precision::I16,
     InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::FP32
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::U64
```

Comment:
Why not an I64? Is there a special use case?

Reply:
No, I'm not aware of a special U64 use case, so will change it to I64.